Libraries/CMSIS/Device/Maxim/MAX32657/Include/max32657.h

```diff
@@ -0,0 +1,907 @@
+    GPIOWAKE_IRQn,          /* 0x2A  0x00A8  42: GPIO Wakeup */
+    CRC_IRQn,               /* 0x2B  0x00AC  43: CRC */
+    AES_IRQn,               /* 0x2C  0x00B0  44: AES */
+    ERFO_IRQn,              /* 0x2D  0x00B4  45: ERFO Ready */
+    BOOST_IRQn,             /* 0x2E  0x00B8  46: Boost Controller */
+    ECC_IRQn,               /* 0x2F  0x00BC  47: ECC */
+    BTLE_TX_DONE_IRQn,      /* 0x30  0x00C0  48: BTLE TX Done */
+    BTLE_RX_RCVD_IRQn,      /* 0x31  0x00C4  49: BTLE RX Received */
+    BTLE_RX_ENG_DET_IRQn,   /* 0x32  0x00C8  50: BTLE RX Energy Detected */
+    BTLE_SFD_DET_IRQn,      /* 0x33  0x00CC  51: BTLE SFD Detected */
+    BTLE_SFD_TO_IRQn,       /* 0x34  0x00D0  52: BTLE SFD Timeout */
+    BTLE_GP_EVENT_IRQn,     /* 0x35  0x00D4  53: BTLE BTLE Timestamp */
+    BTLE_CFO_IRQn,          /* 0x36  0x00D8  54: BTLE CFO Done */
+    BTLE_SIG_DET_IRQn,      /* 0x37  0x00DC  55: BTLE Signal Detected */
+    BTLE_AGC_EVENT_IRQn,    /* 0x38  0x00E0  56: BTLE AGC Event */
+    BTLE_RFFE_SPIM_IRQn,    /* 0x39  0x00E4  57: BTLE RFFE SPIM Done */
+    BTLE_TX_AES_IRQn,       /* 0x3A  0x00E8  58: BTLE TX AES Done */
+    BTLE_RX_AES_IRQn,       /* 0x3B  0x00EC  59: BTLE RX AES Done */
+    BTLE_INV_APB_ADDR_IRQn, /* 0x3C  0x00F0  60: BTLE Invalid APB Address */
+    BTLE_IQ_DATA_VALID_IRQn, /* 0x3D  0x00F4  61:BTLE IQ Data Valid */
+    BTLE_XXXX_IRQn,         /* 0x3E  0x00F8  62: BTLE XXXX TODO(ME30): Verify BTLE IRQs */
+    RSV47_IRQn,             /* 0x3F  0x00FC  63: Reserved */
+    MPC_IRQn,               /* 0x40  0x0100  64: MPC Combined (Secure) */
+    PPC_IRQn,               /* 0x41  0x0104  65: PPC Combined (Secure) */
+    RSV50_IRQn,             /* 0x42  0x0108  66: Reserved  */
+    RSV51_IRQn,             /* 0x43  0x010C  67: Reserved */
+    RSV52_IRQn,             /* 0x44  0x0110  68: Reserved */
+    RSV53_IRQn,             /* 0x45  0x0114  69: Reserved */
+    MXC_IRQ_EXT_COUNT,
+} IRQn_Type;
+// clang-format on
+
+#define MXC_IRQ_COUNT (MXC_IRQ_EXT_COUNT + 16)
+
+/* ================================================================================ */
+/* ==================             Processor Section              ================== */
+/* ================================================================================ */
+
+#define __CM33_REV 0x0000U /**< Cortex-M33 Core revision */
+#define __DSP_PRESENT 1U /**< Presence of DSP  */
+#define __FPU_PRESENT 1U /**< Presence of FPU  */
+#define __MPU_PRESENT 1U /**< Presence of MPU  */
+#define __SAUREGION_PRESENT 1U /**< Presence of FPU  */
+#define __TZ_PRESENT 1U /**< Presence of TrustZone */
+#define __VTOR_PRESENT 1U /**< Presence of VTOR register in SCB  */
+#define __NVIC_PRIO_BITS 4U /**< NVIC interrupt priority bits */
+#define __Vendor_SysTickConfig 0U /**< Is 1 if different SysTick counter is used */
+
+#include <core_cm33.h>
+#include <arm_cmse.h>
+#define IS_SECURE_ENVIRONMENT (defined(__ARM_FEATURE_CMSE) && (__ARM_FEATURE_CMSE == 3U))
```

Comment:
This definition causes issues on zephyr side.
![image](https://github.com/analogdevicesinc/msdk/assets/46590392/0f8dd262-93bd-48ba-9f9c-46ebb069b5ac)

Reply:
@ozersa fixed in https://github.com/analogdevicesinc/msdk/pull/1012/commits/f040b6dea9e524ea9de0506d31fc2328bbf758a2